hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsRestOperation.java

```diff
@@ -478,6 +481,11 @@ private boolean executeHttpOperation(final int retryCount,
       retryPolicy = client.getRetryPolicy(failureReason);
       if (httpOperation instanceof AbfsAHCHttpOperation) {
         registerApacheHttpClientIoException();
+        if (ex instanceof RequestAbortedException
+            && ex.getCause() instanceof IOException
```

Comment:
there should be a specific exception subclass here, switching on exception message is too brittle/ and not the "java style". 

I can't see any ClosedException in the codebase right now, so this is your chance to add one!

can you create such a class in hadoop-common in `org.apache.hadoop.fs` package:

tag as public/unstable.
```
class ClosedIOException extends PathIOException { ... }
```

Reply:
Make sense. Have added the new class ClosedIOException.